src/Cache.php

```diff
@@ -313,4 +340,68 @@ public function getOrSet($key, callable $callable, $ttl = null, Dependency $depe
 
         return $value;
     }
+
+    public function __call($name, $arguments)
+    {
+        return call_user_func_array([$this->handler, $name], $arguments);
+    }
+
+    public function enableKeyNormalization(): void
+    {
+        $this->keyNormalization = true;
+    }
+
+    public function disableKeyNormalization(): void
+    {
+        $this->keyNormalization = false;
+    }
+
+    /**
+     * @param string $keyPrefix a string prefixed to every cache key so that it is unique globally in the whole cache storage.
+     * It is recommended that you set a unique cache key prefix for each application if the same cache
+     * storage is being used by different applications.
+     */
+    public function setKeyPrefix(string $keyPrefix): void
+    {
+        if ($keyPrefix != '' && !ctype_alnum($keyPrefix)) {
+            throw new Exception\InvalidArgumentException('Cache key prefix should be alphanumeric');
```

Comment:
That rule disables `-`, `_` and `.` that are common for cache keys.

Reply:
This code was there before, I didn't add it, should I remove it?

Reply:
No.